src/main/java/io/gingersnapproject/cdc/configuration/Configuration.java

```diff
@@ -0,0 +1,17 @@
+package io.gingersnapproject.cdc.configuration;
+
+import java.util.Map;
+
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithName;
+
+@ConfigMapping(prefix = "gingersnap")
+public interface Configuration {
```

Comment:
Should we make the `Database` and `Cache` optional? So they don't need to be registered in the properties at startup. Also, making them required causes the test to complain, although the application works as expected.

Reply:
IMO they should be mandatory as the db-syncer doesn't function without a Database or a Cache. I can update the test.

Reply:
In fact, the GreetingResource and it's test are useless ... so I think we should just remove those.